standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1053,7 +1054,15 @@ public void abortTxn(AbortTxnRequest rqst) throws NoSuchTxnException, MetaExcept
           }
           raiseTxnUnexpectedState(status, txnid);
         }
-        abortTxns(dbConn, Collections.singletonList(txnid), true, isReplayedReplTxn);
+
+        if (isReplayedReplTxn) {
+          errorMsg = TxnErrorMsg.ABORT_REPLAYED_REPL_TXN;
+        } else if (isHiveReplTxn) {
+          errorMsg = TxnErrorMsg.ABORT_DEFAULT_REPL_TXN;
+        } else {
+          errorMsg = TxnErrorMsg.ABORT_ROLLBACK;
```

Comment:
why don't you set it in the var declaration?

Reply:
Assigned `TxnErrorMsg.NONE` as default. Done.